pkg/server/container_list.go

```diff
@@ -69,13 +69,21 @@ func (c *criService) filterCRIContainers(containers []*runtime.Container, filter
 		return containers
 	}
 
+	sb := filter.GetPodSandboxId()
```

Comment:
Can you add a "why" comment here, my understanding is that `GetPodSandboxId()` could be a short ID

Reply:
sure, done!